Tasks/Romanychev/Task5/task5_1.py

```diff
@@ -0,0 +1,44 @@
+from functools import reduce
```

Comment:
just cannot do a simple sum of two numbers without reduce, totally relatable for me as well